### ISSUE TITLE
AOP create case for contact

### ISSUE THREAD

**Issue by JimMackin** [salesagility/SuiteCRM#172]
Previously adding a contact to a case was enough for them to see it in the portal. It's likely that recent changes have broken this. It would be good to add a check so that if there is no contact created id in the case then all associated contacts can see that case.

**Comment by samus-aran** [comment]
@JimMackin does this still apply?  If related to an Account do not contacts see that case due to that relationship?

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [AOP create case for contact](https://trello.com/c/6vuTVaE7/158-aop-create-case-for-contact)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
